metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -91,6 +93,9 @@ tar -xzf %{SOURCE11} -C %{buildroot}%{metron_home}
 install %{buildroot}%{metron_home}/bin/metron-rest %{buildroot}/etc/init.d/
 install %{buildroot}%{metron_home}/bin/metron-management-ui %{buildroot}/etc/init.d/
 
+# allows node dependencies to be packaged in the RPMs
+npm install --prefix="%{buildroot}%{metron_home}/web/expressjs" --only=production
```

Comment:
We package only production dependencies.  We know that all production dependencies are Apache license compatible.  Some of the development dependencies are not Apache license compatible.